src/components/QuickOpenModal.js

```diff
@@ -383,12 +398,8 @@ export class QuickOpenModal extends Component<Props, State> {
             expanded && items[selectedIndex] ? items[selectedIndex].id : ""
           }
         />
-        {!symbols ||
-          (symbols.functions.length == 0 && (
-            <div className="loading-indicator">
-              {L10N.getStr("loadingText")}
-            </div>
-          ))}
+
+        {this.renderSymLoading(symbols, selectedSource, this.isSymbolSearch())}
```

Comment:
It's okay for this to be `this.renderLoading` and for the function to know which modes we show loading messages for

Reply:
by that do you mean something like.... 
```renderLoading( mode: string ) {
     { symbols, selectedSource } = this.props
      if(mode === symbol) {
            {code for symbol loading}
```

Reply:
by that do you mean something like.... 
```renderLoading( mode: string ) {
     { symbols, selectedSource } = this.props
      if(mode === symbol) {
            {code for symbol loading}
```